gems.rb

```diff
@@ -10,4 +10,8 @@
 group :maintenance, optional: true do
 	gem "bake-modernize"
 	gem "bake-bundler"
-end
\ No newline at end of file
+end
+
+group :development do
+  gem "pry"
```

Comment:
I understand the advantage but this is unrelated to the PR :)

Reply:
Well, it was related this way:  I needed to use `pry` to get some of these specs passing. :)

Are you saying you want to see a separate PR with this, or that you'd rather not have this development dependency at all?

Reply:
I think adding pry as a development dependency is outside of the scope of this PR.

I don’t have a strong opinion about introducing pry. I usually just use `binding.irb` these days. I used to use pry a lot but don’t tend to use it much these days.